azure-pipelines.yml

```diff
@@ -48,32 +38,29 @@ jobs:
 
   steps:
   - template: build/install-dotnet.yml
-  
-  - task: 1ESHostedPoolValidation@1
-  - pwsh: |
-      $packageSuffix = 0
-      if (-not $$(hasTag)) {
-        $packageSuffix = $(buildNumber)
-        Write-Host "No git tag found. Setting packages suffix to '$packageSuffix'"
```

Comment:
I am guessing we don't use package suffix any more.

Reply:
nevermind, i see we directly using buildNumber

Reply:
we do -- it's just added to the 'pack' step (`--version-suffix`)

```
      custom: 'pack'
      ${{ if eq(variables.hasTag, true) }}:
        arguments: '--no-build -c Release -o $(buildOutputDirectory)'
      ${{ else }}:
        arguments: '--no-build -c Release -o $(buildOutputDirectory) --version-suffix -$(buildNumber)'
```

Reply:
yeah, i just saw that a little later.